mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -64,7 +64,7 @@ CLASS agf net/minecraft/entity/Entity
 	METHOD B_ getPistonBehavior ()Lcdf;
 	METHOD C_ shouldBroadcastConsoleToOps ()Z
 	METHOD E updatePortalCooldown ()V
-	METHOD Q getDisplayedName ()Lja;
+	METHOD Q getDisplayName ()Lja;
```

Comment:
`Display` over `Displayed` as it's what will be be used as the display name assuming nothing changes it, not the currently displayed name.